### ISSUE TITLE
Add traditional layout example

### ISSUE THREAD

**Issue by davbeck** [davbeck/TURecipientBar#2]
In addition to the auto layout example, there should be an example of how someone would use the bar in a traditional layout environment.

**Comment by alonecuzzo** [comment]
Yeah I agree with @xzenon, there's too much hidden in IB/storyboards... would be nice to see just a vanilla code version where it shows how to set stuff up.  With code there's no hidden drop downs etc etc. to slodge through; it's all just right there.
